### ISSUE TITLE
Create nodes iterator

### ISSUE THREAD

**Issue by jimboca** [UniversalDevicesInc/udi_python_interface#3]
Previously could just do this:
```
        for node in self.nodes:
            if self.nodes[node].address != self.address:
                self.nodes[node].query()
```
But now have to call a method and define our own list of nodes
```
        nodes = self.poly.getNodes()
        for node in nodes:
            if nodes[node].address != self.address:
                nodes[node].query()
```
should be done the real python way? https://docs.python.org/3/library/itertools.html

**Comment by bpaauwe** [comment]
Agree that there should be a better way to iterate over the node list.  However, I don't think access the internal node list data structure directly from the interface module is the right way to do it.

**Comment by jimboca** [comment]
Yes, agree, only supplied that for info, it's actually a problem now because PGC does it differently.

**Comment by Goose66** [comment]
In my Nodeserver wrapper class I have two properties: "nodes" which is an iterable collection of Node objects, and "nodesDict" which is an iterable dictionary of address: node pairs. They return  self.poly.nodes_internal.values() and self.poly.nodes_internal respectively.
